src/rosdep2/sources_list.py

```diff
@@ -487,9 +487,20 @@ def update_sources_list(sources_list_dir=None, sources_cache_dir=None,
     python_versions = {}
 
     print('Query rosdistro index %s' % get_index_url())
-    for dist_name in sorted(get_index().distributions.keys()):
+    distribution_name_list = sorted(get_index().distributions.keys())
```

Comment:
I would suggest to remove the `sorted()` from this line and instead only perform it in the `for` loop below since the next line doesn't need the list to be sorted. I would also suggest to name the variable `distribution_names` (since it is not necessarily a list but a generator / dict keys).

Reply:
sounds good https://github.com/ros-infrastructure/rosdep/pull/738/commits/e61e13ff3f97e837444f5036e2bf6c5ab9c26168